docs/source/general/setup.rst

```diff
@@ -210,24 +231,24 @@ Add and modify the following lines accordingly:
     # (eg: for en event) and have an expiration date set.
     30 04 * * * <virtualenv_path>/bin/python <full/path/to>/manage.py deactivate_expired_users
 
-    # This command deletes users that have expired (and should have 
-    # been deactivated by deactivate_expired_users) for more than 
+    # This command deletes users that have expired (and should have
+    # been deactivated by deactivate_expired_users) for more than
     # 18 months (which is the default duration)
     30 04 * * * <virtualenv_path>/bin/python <full/path/to>/manage.py delete_old_users
 
 Be sure to replace ``<virtualenv_path>`` with the full path to the Python
-virtual environment. 
+virtual environment.
 
 Also, change ``<full/path/to>`` to the directory where ``manage.py`` is.
 
-To get the full path to ``manage.py`` when django-freeradius is 
-installed for development, navigate to the base directory of 
+To get the full path to ``manage.py`` when django-freeradius is
```

Comment:
The use of `absolute path` instead of `full path` seems more appropriate.

Reply:
👍